app/assets/stylesheets/app/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  border-radius: 2px;
+  background-color: $white;
+  font-family: $main-font;
+
+  &__title {
+    height: 48px;
```

Comment:
Properties should be ordered color, font-weight, height, line-height, text-align